---
string[] item_names     # List of item names (e.g., ["coffee", "beer", "soda"])